helm/content-store/templates/deployment.yaml

```diff
@@ -16,9 +16,9 @@ spec:
     spec:
       containers:
         - name: {{ .Release.Name }}
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+          image: "{{ .Values.appImage.repository }}:{{ .Values.appImage.tag }}"
           ports:
-            - containerPort: {{ .Values.containerPort }}
+            - containerPort: {{ .Values.appPort }}
```

Comment:
I'd recommend always specifying a port name ([example](https://github.com/bitnami/charts/blob/f88d9fae4eaeba918df2aec7a76efb268e3d0318/bitnami/nginx/templates/deployment.yaml#L157)), so that health check, Services etc. that reference the port don't need to be updated if the containerPort is changed.

Reply:
Whats your thoughts on the following name - this port will be local port for the Rails app conatiner ( - name: {{ .Release.Name }}-port )

Reply:
in a way it is a http port - on the rails side so could also be ( -name : http ) baring it uses port 3000 but i could change the port to 8080

Reply:
It should be `http` as that's the protocol the port is exposing.